ci-scripts/publish-rc.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+rm -rf dist
```

Comment:
so I think the build is actually happening before this - we should add it to the travis config - we could swap out `yarn build:ui` for `yarn build` [here](https://github.com/SAP/fundamental-vue/blob/master/.travis.yml#L13). I believe this would achieve what we're looking for, although adding a small amount of wait time. We can do another iteration later to improve travis build times once we get all the pieces working together correctly.

Reply:
Another option is to create an additional build script that just does these parts and put it below the push here- see what we did in ngx with `npm run build-deploy-library` [here](https://github.com/SAP/fundamental-ngx/blob/master/ci-scripts/publish-rc.sh#L18). If we choose to do it this way, it will need to be added to `ci-scripts/publish.sh` as well.

Reply:
I tend to agree with @jbadan. Putting something like this in `publish-rc.sh` (and `publish.sh`) seems wrong.  Although it would work, it adds some "mystery" steps to these scripts.  I like the idea of either adding it to `.travis.yml` or making a different script target to handle what is necessary.

Reply:
Using `yarn build` instead of `yarn build:ui` does not work @jbadan. If you run `yarn build:ui` only the UI components are built. Webpack will output a summarized report after the build is complete:

<img width="503" alt="screenshot 2019-02-25 at 08 01 17" src="https://user-images.githubusercontent.com/153225/53319900-40a09a00-38d4-11e9-97b3-b978480321e1.png">

As you can see when running `yarn build:ui` the genrated umd build is *only* ~25kb in size (gziped). When you run `yarn build` you will get something different:

<img width="532" alt="screenshot 2019-02-25 at 08 03 09" src="https://user-images.githubusercontent.com/153225/53319947-65950d00-38d4-11e9-9602-5e3291449048.png">

I am not 100% as to why there is such a huge difference. I assume that webpack/babel is adding additional polyfills and other dependencies to the build because it sees that those are needed because the documentation app needs those things...

The main idea behind this strict separation (having a separate ui project and a separate docs project) was to reliably be able to generate production builds of Fundamental Vue without any magic being involved. In the past we had problems with polyfills that somehow made it into Fundamental Vue - by mistake.

Reply:
When it comes to those things I am a total noob. :D

I just want to make sure we are on the same page on this so correct me if I am wrong but before we release anything we want to ensure a few things:

- Can everything be built? Everything means:
  - Fundamental Vue Components on it's own: `yarn build:ui`
  - Documentation App: `yarn build:docs`
  - Everything together (which is used during development): `yarn build`
- Are the tests working? `yarn test`
- Is the linter happy? `yarn lint`

If we get a *yes* to all those questions then the Fundamental Vue UI Components should be released. This can happen by running `yarn build:ui`. You are saying that what I added to the script in this file should actually go to the travis configuration - right? I will try to make that change today. If it turns out to be not what you meant may I suggest that I stay late sometime this week so that we can work on this together? :)